settings.gradle.kts

```diff
@@ -100,13 +100,14 @@ gradleEnterprise {
 }
 
 gitRepositories {
-  checkoutsDirectory.set(file(rootProject.projectDir.absolutePath + "/hedera-node/hapi/"))
+  checkoutsDirectory.set(File(rootDir, "hedera-node/hapi"))
+  // check branch in repo for updates every 10 minutes
+  refreshIntervalMillis.set(10 * 60 * 1000)
   include("hedera-protobufs") {
     uri.set("https://github.com/hashgraph/hedera-protobufs.git")
     // choose tag or branch of HAPI you would like to test with
     // this looks for a tag in hedera-protobufs repo
-    // This version needs to match tha HAPI version below in versionCatalogs
-    tag.set("add-pbj-types-for-state")
+    branch.set("add-pbj-types-for-state")
```

Comment:
The reason I was using tag.set was it worked for a tag or branch I think. I remember looking at the plugin source code. During development we will want branches and then tags for release.

Reply:
You are right. I double checked the plugin code. Should not make a difference -> reverted.

Then I wondered why it does not work. On thing is the "refreshIntervalMillis" (see above) which defaults to 24h. I changed it to update every second just to be sure. Then we can observe. The git-include-plugin contains the code to pull the latest changes. And my testing indicates that it works.

I think there is an issue in the `com.hedera.pbj.pbj-compiler` plugin. After changes from the repo were pulled, the `generatePbjSource` was still `UP-TO-DATE`. And then the code was not re-generated and you get compile errors. I'll check the plugin and see if I can find the issue there.